frontend/src/resources/provider-connection.ts

```diff
@@ -73,6 +73,8 @@ export interface ProviderConnectionStringData {
   pullSecret?: string
   ['ssh-privatekey']?: string
   ['ssh-publickey']?: string
+  kubeconfig?: string
+  externalInfraNamespace?: string
```

Comment:
Let's group these and add a comment to say which provider they are for (OpenShift Virtualization).